### PR TITLE
Update dependecies

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: src/modules/rabbitmq/blockchain-events/nft-events.consumer.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export class NftEventsConsumer {
     dlqExchange: process.env.RABBITMQ_DLQ_EXCHANGE,
   })
   async consumeAuctionEvents(nftAuctionEvents: any) {
+    console.log(11111111, nftAuctionEvents)
```
Comment (giurgiur99) [inline src/modules/rabbitmq/blockchain-events/nft-events.consumer.ts]: I guess this should be removed

Comment (bogdan-rosianu) [inline src/modules/rabbitmq/blockchain-events/nft-events.consumer.ts:36]: remove this

Comment (danielailie) [reply src/modules/rabbitmq/blockchain-events/nft-events.consumer.ts]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by giurgiur99 — APPROVED** [review]

File: src/modules/rabbitmq/blockchain-events/nft-events.consumer.ts (hunk 1 above)
Comment (danielailie) [reply src/modules/rabbitmq/blockchain-events/nft-events.consumer.ts]: done

**Pull request merged**